src/QMCWaveFunctions/Jastrow/DiffTwoBodyJastrowOrbital.h

```diff
@@ -139,7 +145,18 @@ class DiffTwoBodyJastrowOrbital : public DiffWaveFunctionComponent
         lapLogPsi[i]  = new ValueVectorType(NumPtcls);
       }
       OffSet.resize(F.size());
-      int varoffset = myVars.Index[0];
+
+      // Find first active variable for the starting offset
+      int varoffset = -1;
+      for (int i = 0; i < myVars.size(); i++)
+      {
+        varoffset = myVars.Index[i];
+        if (varoffset != -1)
+        {
```

Comment:
Please remove {} around break to save two lines.

Reply:
fixed